tools/bin/e2e_test.sh

```diff
@@ -16,11 +16,15 @@ mkdir -p /tmp/airbyte_local
 
 # Detach so we can run subsequent commands
 VERSION=dev TRACKING_STRATEGY=logging docker-compose up -d
-trap 'echo "docker-compose logs:" && docker-compose logs -t --tail 1000 && docker-compose down && docker rm -f $(docker ps -q --filter name=airbyte_ci_pg)' EXIT
+trap 'echo "docker-compose logs:" && docker-compose logs -t --tail 1000 && docker-compose down && docker stop airbyte_ci_pg' EXIT
 
-docker run -d -p 5433:5432 -e POSTGRES_PASSWORD=secret_password -e POSTGRES_DB=airbyte_ci --name airbyte_ci_pg postgres
-echo "Waiting for services to begin"
-sleep 30 # TODO need a better way to wait
+docker run --rm -d -p 5433:5432 -e POSTGRES_PASSWORD=secret_password -e POSTGRES_DB=airbyte_ci --name airbyte_ci_pg postgres
```

Comment:
:information_source: Using `--rm` here to automatically remove the container one stopped, allows to shorten the above `trap` command to a way simpler `stop`.